src/main/cpp/t09_row.cpp

```diff
@@ -23,5 +23,27 @@
 using namespace std;
 
 int t09_row() {
+ int a = 0 ;
+ int b = 1;
+ int c = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'c' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4349150)